app/models.py

```diff
@@ -18,6 +18,12 @@ def __init__(self, email, password):
     def __repr__(self):
         return '<User {}: {}>'.format(self.id, self.email)
 
+    def serialize(self):
```

Comment:
'to_json' would be more correct than 'serialize', since serialization implies binary encoding.

Reply:
Actually, I suppose serialization doesn't necessarily imply binary, but if we do add binary serialization later e.g. for caching, it would be useful to keep the method names clearer.

Reply:
Oooh, interesting. I was trying to be format-agnostic, but I think that makes sense. Gonna go w/ to_dict since it's technically not json yet